src/components/event/eventCreate/mutations/EventCreateMutation.tsx

```diff
@@ -0,0 +1,32 @@
+import { graphql } from "react-relay";
+import { ROOT_ID, SelectorStoreUpdater } from "relay-runtime";
+
+import { connectionUpdater } from "../../../../relay";
+
+export const EventCreate = graphql`
+  mutation EventCreateMutation($input: EventCreateInput!) {
+    EventCreate(input: $input) {
+      event {
+        id
+        name
+        start
+        end
+        allDay
+      }
+      error
+      success
+    }
+  }
+`;
+
+export const updater: SelectorStoreUpdater = (store) => {
```

Comment:
use the new directives from relay to update connections https://relay.dev/docs/guided-tour/list-data/updating-connections/